README.md

```diff
@@ -35,8 +40,10 @@ or apt to install these components if they're self-hosted.
 
 ### What splunk affects
 
-* Installs the Splunk/Forwarder package and manages their config files. It does not purge them by default.
-* The module will set up both Splunk and Splunkforwarder to run as the 'root' user on POSIX platforms.
+* Installs the Splunk/Forwarder package and manages their config files. It does
+  not purge them by default.
+* The module will set up both Splunk and Splunkforwarder to run as the 'root'
+  user on POSIX platforms.
```

Comment:
Adding three words to a sentence means I then need to manually adjust all the line spacing for the rest of the paragraphs.... yeah I know there are editor plugins that can do this, I just don't see the point in enforcing this - I quite like text to wrap to the length I choose by resizing my terminal window.... maybe it's just me ;-)